### PR TITLE
sig-docs: promote breeswish to committer

### PR DESCRIPTION
**Pull request by ran-huang**
Signed-off-by: Ran <huangran@pingcap.com>

@breeswish has contributed a lot to Docs SIG. According to [Be promoted to Committer
](https://github.com/pingcap/community/blob/master/special-interest-groups/sig-docs/roles-and-organization-management.md#be-promoted-to-committer), we would like to promote him to a new committer of Docs SIG. His detailed contribution is shown below:
1. Have [reviewed 46 PRs in docs repo](https://github.com/pingcap/docs-cn/pulls?q=is%3Apr+reviewed-by%3Abreeswish+merged%3A%3E%3D2019-11) within the past one year
2. Helped [resolve 1 issue](https://github.com/pingcap/docs-cn/issues/1795)  
3. [Added a script to verify link anchors](https://github.com/pingcap/docs-cn/pull/3835) in the docs-cn repo (a big feature)
4. Sponsored by two incumbent committers: @ran-huang, @TomShawn

### CODE REVIEW

File: special-interest-groups/sig-docs/membership.md
Comment (lilin90) [inline special-interest-groups/sig-docs/membership.md:20]: I agree with promoting breeswish & amyangfei to committers this time and expect other candidates to have a better command of both Chinese and English technical writing and documentation rules.

Comment (ran-huang) [reply special-interest-groups/sig-docs/membership.md:19]: Updated~

**Comment by hi-rustin** [comment]
Please waiting this issue https://github.com/tidb-community-bots/ti-community-bot/issues/107 fixed.

**Review by lilin90 — APPROVED** [review]
LGTM

**Comment by hi-rustin** [comment]
> Please waiting this issue [tidb-community-bots/ti-community-bot#107](https://github.com/tidb-community-bots/ti-community-bot/issues/107) fixed.

Already fixed. @ran-huang  Please resolve the conflicts.

**Comment by hi-rustin** [comment]
@pingcap/tidb-maintainer 

Only the maintainer can approve the PR for adding a committer.
About the permission design you can refer to [this doc](https://docs.google.com/document/d/1gXQ5Qglgm2nhyU08ZVeluBM3PBKGgfN5zhgyalqhdyY/edit).

And about the `ti-community-prow-bot` you can refer to [the prow bot book](https://book.prow.tidb.io/).

**Comment by ran-huang** [comment]
@AndreMouche PTAL. This PR needs LGTM from maintainers.

**Comment by AndreMouche** [comment]
> @AndreMouche PTAL. This PR needs LGTM from maintainers.

Could you please explain the reason for example https://github.com/pingcap/community/pull/330

**Comment by ran-huang** [comment]
@breeswish has contributed a lot to Docs SIG. According to [Be promoted to Committer
](https://github.com/pingcap/community/blob/master/special-interest-groups/sig-docs/roles-and-organization-management.md#be-promoted-to-committer), we would like to promote him to a new committer of Docs SIG. His detailed contribution is shown below:
1. Have [reviewed 46 PRs in docs repo](https://github.com/pingcap/docs-cn/pulls?q=is%3Apr+reviewed-by%3Abreeswish+merged%3A%3E%3D2019-11) within the past one year
2. Helped [resolve 1 issue](https://github.com/pingcap/docs-cn/issues/1795)  
3. [Added a script to verify link anchors](https://github.com/pingcap/docs-cn/pull/3835) in the docs-cn repo (a big feature)
4. Sponsored by two incumbent committers: @ran-huang, @TomShawn

@AndreMouche PTAL. Do we need detailed reasons for active contributors?

**Review by winkyao — COMMENTED** [review]
LGTM

**Comment by hi-rustin** [comment]
@winkyao  You should approve it not just comment.

About `LGTM`, you can refer to https://book.prow.tidb.io/plugins/lgtm.html#%E5%AE%9E%E7%8E%B0%E6%80%9D%E8%B7%AF

**Review by winkyao — APPROVED** [review]

**Review by jackysp — COMMENTED** [review]
LGTM

**Pull request merged**
